ddtrace/tracer/option_test.go

```diff
@@ -190,3 +190,16 @@ func TestServiceName(t *testing.T) {
 		assert.Equal("", globalconfig.ServiceName())
 	})
 }
+
+func TestGlobalTag(t *testing.T) {
+	t.Run("InStatsTags", func(t *testing.T) {
+		assert := assert.New(t)
+		tracer := newTracer(
+			WithGlobalTag("k", "v"),
+		)
+		c := tracer.config
+
+		assert.Equal("v", c.globalTags["k"])
+		assert.Contains(statsTags(c), "k:v")
```

Comment:
Let's make this function shorter.
```suggestion
func TestStatsTags(t *testing.T) {
	var c config
	WithGlobalTags("k", "v")(&c)
	assert.Contains(t, statsTags(&c), "k:v")
}
```
We don't need to test that `WithGlobalTags` works. That's for another test.

Reply:
👍 good point, fixed